libcoap/src/context.rs

```diff
@@ -406,6 +410,32 @@ impl CoapContext<'_> {
         self.add_endpoint(addr, coap_proto_t_COAP_PROTO_DTLS)
     }
 
+    // TODO: Possibly wrap in feature?
+    /// Joins a multicastgroup
```

Comment:
The documentation for this method should at least explain the meaning of `groupname` and `ifname` (ideally also explaining that in most cases, `groupname` will just be a multicast IP address).